### ISSUE TITLE
Add ability to change a stream's output on the Streamer

### ISSUE THREAD

**Issue by Lohrer** [micro-nova/AmpliPi#617]
On the full amplified AmpliPi zones can be swapped by clicking one of the current zones, then once the pop-up appears changing the selection of checkboxes:
![Screenshot_20240109_134305](https://github.com/micro-nova/AmpliPi/assets/6646216/f39eab24-bd6e-4fed-a78e-82bc66396d6a)

On a streamer clicking on the "Output X" tile does nothing, it would be great to be able to have the same pop-up appear as on the full AmpliPis. 
![Screenshot_20240109_131815](https://github.com/micro-nova/AmpliPi/assets/6646216/b0c10881-c9f7-42ca-a528-b789fc860e6c)

**Comment by SteveMicroNova** [comment]
I agree, I'm tossing this at the project board and we'll see what (if any) release this makes it into after a meeting
